src/__tests__/3box.test.js

```diff
@@ -247,7 +239,7 @@ describe('3Box', () => {
 
   afterAll(async () => {
     await testUtils.stopIPFS(ipfs, 0)
-    await testUtils.stopIPFS(ipfsBox, 1)
+    return testUtils.stopIPFS(ipfsBox, 1)
```

Comment:
I don't get why using await wouldn't work. I thought async functions are converted to promises that resolve when the last await is done?

Reply:
yeah, tbh not sure what jest is doing with last line awaits, but in idWallet.integration.js the afterAll function was running before the test finished. This worked, i think i also saw a similar problem in the past. 

It was mostly for idWallet tests to pass, so could also not changes lines in other tests right now. It was clearly happen in idWallet, maybe not the others, although they seemed to pass a little more regularly. Still get enough random failures though

Reply:
Ok, let's keep it like this. It's not really wrong :)